dots/.scripts/getmovies

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash -e
+
+# Script to fetch movies *aarrgs*
+
+# Usage:
+# getmovies dragon broly (the movies metadata)
+# getmovies -q dragon broly (get just the id of the movies)
+
+# Options:
+#     -m      Movies metadata
+#     -q      Movies ID(s)
+#     -v      Open vlc player for the first movie (If it finds more than one)
```

Comment:
It will only play movies under the domain of `https://storage.googleapis.com`.

If you encounter a movie on another domain for example: `https://onedrive.live.com` I don't know what is going to happen. 🤷‍♂️ 

I'll validate this later so hiroshima don't occur on this script. (: